tuftool/tests/root_command.rs

```diff
@@ -0,0 +1,274 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+mod test_utils;
+
+use assert_cmd::Command;
+
+#[ignore]
+#[test]
+// Ensure we can create and sign a root file
+fn create_root() {
+    let key = test_utils::test_data().join("snakeoil.pem");
+    // Create root.json
+    Command::cargo_bin("tuftool")
+        .unwrap()
+        .args(&["root", "init", "tests/root_test/root.json"])
```

Comment:
We usually use `TempDir::new` which deletes itself after the test. Best to always cleanup when creating files during tests.

```rust
        .args(&["root", "init", "tests/root_test/root.json"])
```

Reply:
The root subcommands of tuftool use a relative path, so I don't think TempDir would work.

Reply:
The new commit addresses this change.